lib/src/coord/coord.dart

```diff
@@ -9,8 +9,8 @@ import 'package:graphic/src/dataflow/tuple.dart';
 import 'package:graphic/src/graffiti/scene.dart';
 import 'package:graphic/src/util/assert.dart';
 
-import 'rect.dart';
 import 'polar.dart';
+import 'rect.dart';
```

Comment:
these were due to my auto saving formatter. I'll rollback if we move forward with this PR.